ILRepack/RepackOptions.cs

```diff
@@ -79,8 +78,10 @@ public List<string> AllowedDuplicateNameSpaces
         private readonly RepackAssemblyResolver globalAssemblyResolver = new RepackAssemblyResolver();
         private List<Regex> excludeInternalizeMatches;
 
-        public void SetSearchDirectories(string[] dirs)
+        public void SetSearchDirectories(IEnumerable<string> dirs)
         {
+            if (TargetPlatformDirectory != null)
+                dirs = new[] { TargetPlatformDirectory }.Concat(dirs);
```

Comment:
In v4.5's case (at least one mono), I think we need to also include the directory "Facades".

Tried @biz0n's testcase and still got an error
```
$ mono ILRepack.exe /targetplatform:v4,/Library/Frameworks/Mono.framework/Versions/4.0.0/lib/mono/4.5 /out:test.dll Autofac.dll Nito.AsyncEx.dll
INFO: Adding assembly for merge: Autofac.dll
INFO: Adding assembly for merge: Nito.AsyncEx.dll
ERROR: Failed to load assembly Nito.AsyncEx.dll
Mono.Cecil.AssemblyResolutionException: Failed to resolve assembly: 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
...
```
If I'd specify the "Facades" directory, however, it would work fine (except for some missing field in ResourceWriter - since the Mono implementation is different than MS's one :( )

This is the listing for v4.5's mono lib: http://pastebin.com/8v6nqXt9
As you can see we've got *System.Runtime* in `Facades`